### ISSUE TITLE
Send queue is broken. Suspect similar to issue #43

### ISSUE THREAD

**Issue by pandoras-daddy** [bdsmlr-development/bdsmlr-updates#45]
![Error](https://user-images.githubusercontent.com/89096188/129769420-581941e5-b165-4796-9633-a38e42a7c086.png)
All attempts at using the queue fail with each pic having a white cross through them.
Hovering over the pic reveals a lot of server-side error code. See edited screencap attached

**Comment by jenevieveee** [comment]
Yes, it is the same issue. The code responsible for uploading images is the same ( /savephotos endpoint ) regardless of whether it's through the blog, mass uploaders, or whatever.
